DiamondLightSource/SynchWeb
# LIMS-1278: Remove EM from SynchWeb

**JIRA ticket**: [LIMS-1278](https://jira.diamond.ac.uk/browse/LIMS-1278)

**Summary**:

This pull request removes EM related resources from SynchWeb, and replaces existing data collection pages with calls to redirect to PATo.

**Changes**:
- Removal of a significant number of EM related files
- Deletion of some references to EM in other files
- Data collection page is now replaced by a redirect
- New `config.php` field, "$redirects"

**To test**:
- Visit a MX collection list and check that all data is displayed normally (https://localtest.diamond.ac.uk/dc/visit/cm37235-2)
- Check data is displayed as normal for other beamlines, such as i19-1 (https://localtest.diamond.ac.uk/dc/visit/cm37266-1), i15-1 (https://localtest.diamond.ac.uk/dc/visit/cm37261-1) and VMXi (https://localtest.diamond.ac.uk/dc/visit/nt37104-68)
- Load visit stats page (https://localtest.diamond.ac.uk/stats/visit/cm37235-2), guarantee all is loaded
- Visit report pages (https://localtest.diamond.ac.uk/stats/overview/beamlines and https://localtest.diamond.ac.uk/stats/overview/beamlines) and confirm the content is there
- Visit an EM visit (https://localtest.diamond.ac.uk/dc/visit/bi23047-119) and check that the redirect message appears
